package.json

```diff
@@ -16,11 +14,14 @@
     "@mui/material": "^5.3.1",
     "@mui/system": "^5.4.3",
     "@popperjs/core": "2.5.1",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "12.1.2",
     "axios": "^0.25.0",
     "chart.js": "2.9.3",
     "d3": "^7.3.0",
     "gulp": "4.0.2",
     "gulp-append-prepend": "1.0.8",
+    "js-file-download": "^0.4.12",
```

Comment:
@timAmida for future reference...
1) Please avoid adding additional packages into projects. Check with everyone first.
2) This package isn't ideal because it hasn't been updated or published to for 2 years. That's a loooong period of time to go without maintenance. 

I'm going to remove this and replace it with a ticket to do this ourselves.